docs/modules/ROOT/pages/release-notes-5.4.1.adoc

```diff
@@ -0,0 +1,11 @@
+= 5.4.1 Release Notes
+
+== New Features
+
+There are no new features in this release.
+
+== Fixes
+
+* Custom template actions aren't loaded when a template is cloned initially.
```

Comment:
Does this mean:

> Custom template actions are not initially loaded when a template is cloned

Or:

> Custom template actions aren't loaded when a template is initially loaded

?

Reply:
It's "Custom template actions are not initially loaded when a template is cloned"

Reply:
```suggestion
* Custom template actions aren't initially loaded when a template is cloned.
```